docs/ldml/tr35-keyboards.md

```diff
@@ -1387,9 +1400,17 @@ Layers are matched exactly based on the modifier keys which are down. For exampl
 
 - `other` as a modifier will match if no other layers match.
 
-Multiple modifier sets may be separated by commas.  For example, `none, shift caps` will match either no modifiers *or* shift and caps.  `ctrlL altL, altR` will match either  left-control and left-alt, *or* right-alt.
+Multiple modifier sets are separated by commas.  For example, `none, shift caps` will match either no modifiers *or* shift and caps.  `ctrlL altL, altR` will match either  left-control and left-alt, *or* right-alt.
+
+Keystrokes must be ignored where there isn’t a layer that explicitly matches nor a layer with `other`. Example: If there is a `ctrl` and `shift` layer, but no `ctrl shift` nor `other` layer, no output will result from `ctrl shift X`.
+
+Layers are not allowed to overlap in their matching.  For example, the keyboard author will receive an error if one layer specifies `alt shift` and another layer specifies `altR shift`.
+
+There is one special case:  the `other` layer matches if and only if no other layer matches. Thus logically the `other` layer is matched after all other layers have been checked.
+
+Because there is no overlap allowed between layers, the order of `<layer>` elements is not significant.
 
-Keystrokes where there isn’t an explicitly matching layer, and where there is no layer with `other` specified, are ignored.
+> Note: The modifier syntax may be enhanced in the future. but will remain backwards compatible with the syntax described here.
```

Comment:
```suggestion
> Note: The modifier syntax may be enhanced in the future, but will remain backwards compatible with the syntax described here.
```

Reply:
Huh? It is correcting an obvious typo.

Reply:
That change was accepted in https://github.com/unicode-org/cldr/pull/3497/commits/0982276d53b3594e4b74848822e8f407a4db157f